<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sheldon</string>
  </property>
  <property name="windowIcon">
   <iconset resource="sheldon.qrc">
    <normaloff>:/resource/icons/appicon.png</normaloff>:/resource/icons/appicon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>651</width>
      <height>421</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::South</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Configurar</string>
     </attribute>
     <widget class="QTextEdit" name="mirrortodownload">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>621</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="pathtosave">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>321</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>531</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Mirrors a descargar (Debes especificar la arquitectura, deb-i386, deb-amd64, etc): </string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Guardar en:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBrowse">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>30</y>
        <width>92</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Examinar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnApply">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>350</y>
        <width>92</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Aplicar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>200</y>
        <width>481</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Descargar</string>
     </attribute>
     <widget class="QTextEdit" name="output">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>511</width>
        <height>351</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>62</red>
            <green>186</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>62</red>
            <green>186</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>118</red>
            <green>118</green>
            <blue>117</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Terminal</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnStart">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>92</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Iniciar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnStop">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>92</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Detener</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConfigurar"/>
   <addaction name="actionAcerca_de"/>
   <addaction name="actionSalir"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConfigurar">
   <property name="icon">
    <iconset resource="sheldon.qrc">
     <normaloff>:/resource/icons/galternatives.png</normaloff>
     <normalon>:/resource/icons/galternatives.png</normalon>:/resource/icons/galternatives.png</iconset>
   </property>
   <property name="text">
    <string>Configurar</string>
   </property>
   <property name="toolTip">
    <string>Configurar</string>
   </property>
  </action>
  <action name="actionAyuda">
   <property name="icon">
    <iconset resource="sheldon.qrc">
     <normaloff>:/resource/icons/help-contents.png</normaloff>
     <normalon>:/resource/icons/help-contents.png</normalon>:/resource/icons/help-contents.png</iconset>
   </property>
   <property name="text">
    <string>Ayuda</string>
   </property>
   <property name="toolTip">
    <string>Ayuda</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="icon">
    <iconset resource="sheldon.qrc">
     <normaloff>:/resource/icons/exit.png</normaloff>:/resource/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="toolTip">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="icon">
    <iconset resource="sheldon.qrc">
     <normaloff>:/resource/icons/info.png</normaloff>:/resource/icons/info.png</iconset>
   </property>
   <property name="text">
    <string>Acerca de</string>
   </property>
   <property name="toolTip">
    <string>Acerca de</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sheldon.qrc"/>
 </resources>
 <connections/>
</ui>
